eth/contracts/StableSwap/StableSwapTwoPool.sol

```diff
@@ -359,7 +401,13 @@ contract StableSwapTwoPool is Ownable, ReentrancyGuard {
         // Mint pool tokens
         token.mint(msg.sender, mint_amount);
 
-        emit AddLiquidity(msg.sender, amounts, fees, D1, token_supply + mint_amount);
+        emit AddLiquidity(
+            msg.sender,
+            amounts,
+            fees,
+            D1,
+            token_supply + mint_amount
+        );
     }
 
     function get_y(
```

Comment:
We won't touch this `i, `j` .. here? This is the function we'll call in MOST contract?

Reply:
In most we will call exchange, in the frontend we will cal one of this or get_dy.

Reply:
changed this in get_dy and exchange function. Now its mixed up trhough the file but consistent on the subset we will use. Not sure if we want to change everything?

Reply:
I'm good now thanks